### PR TITLE
[EA Forum only] allow admins to edit other users profiles

### PR DESCRIPTION
**Pull request by s-cheng**
This PR provides a way for admins to edit other users' profiles.

EDIT: Ok, decided to go ahead and move the "Message" button up. All this is still EA Forum only.

Screen Shot 2022-05-10 at 6 38 25 PM

-----
~~Unfortunately now there are too many links here, which causes one to wrap. I was hoping this would be quick, but I think I'll have to figure out how to update this part of the page to fix it. Maybe I will just go ahead and move the "Message" button above the rest.~~

Screen Shot 2022-05-10 at 5 37 10 PM

### CODE REVIEW

File: packages/lesswrong/components/users/UsersProfile.tsx [hunk 1]
```diff
@@ -44,16 +45,10 @@ const styles = (theme: ThemeType): JssStyles => ({
     paddingRight: 10,
     marginLeft: "auto",
     [theme.breakpoints.down('md')]: {
-      ...(forumTypeSetting.get() === "EAForum" ? {
-        gridTemplateColumns: `${SECTION_WIDTH}px 1fr`,
-        gridTemplateAreas: `
-          'center right'
-        `,
-      } : {}),
+      display: 'block',
```
Comment (s-cheng) [inline packages/lesswrong/components/users/UsersProfile.tsx:48]: Also decided to keep the middle column centered for all screen widths, and just move the right sidebar info under the bio at mid widths.

**Comment by Oldmantax** [comment]
I wonder if we should rename the "edit account" option to "account settings" - to me that better explains what that option actually is, and I also think it makes clearer how "edit account" and "edit profile" differ.

File: packages/lesswrong/components/users/EditProfileForm.tsx [hunk 2]
```diff
@@ -30,25 +30,39 @@ const EditProfileForm = ({classes}: {
...
   const { history } = useNavigation()
+  const { params } = useLocation()
   
-  const { Typography, WrappedSmartForm } = Components;
+  const { Typography, WrappedSmartForm } = Components
   
-  if (!currentUser) {
+  const terms: {slug?: string, documentId?: string} = params.slug ?
+    { slug: params.slug } :
+    currentUser ? { documentId: currentUser._id } : {}
```
Comment (jpaddison3) [inline packages/lesswrong/components/users/EditProfileForm.tsx:39]: I would avoid this nested ternary
```suggestion
  let terms: {slug?: string, documentId?: string} = {}
  if (params.slug) {
    terms.slug = params.slug
  } else if (currentUser) {
    terms.documentId = currentUser._id
  }
```

File: packages/lesswrong/components/users/UsersProfile.tsx (hunk 1 above)
Comment (jpaddison3) [reply packages/lesswrong/components/users/UsersProfile.tsx:48]: Good choice

**Review by jpaddison3 — APPROVED** [review]
Nice, thanks

**Comment by Oldmantax** [comment]
FYI @s-cheng I think you didn't intend "Account settings" to be on a new line in the below. But maybe this is an admin-only bug.
image

**Pull request merged**